src/purchases.ts

```diff
@@ -777,6 +777,45 @@ export default class Purchases {
         RNPurchases.setMparticleID(mparticleID);
     }
 
+    /**
+     * Subscriber attribute associated with the CleverTap Id for the user
+     * Required for the RevenueCat CleverTap integration
+     *
+     * @param cleverTapID Empty String or null will delete the subscriber attribute.
```

Comment:
I think it's pretty self-explanatory but maybe add something about what the param will do normally? Same for the other methods.
```suggestion
     * @param cleverTapID CleverTap user ID to use in CleverTap integration. Empty String or null will delete the subscriber attribute.
```